sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -740,10 +740,11 @@ object RapidsConf {
 
   val CLOUD_SCHEMES = conf("spark.rapids.cloudSchemes")
     .doc("Comma separated list of additional URI schemes that are to be considered cloud based " +
-      "filesystems. Schemes already included: dbfs, s3, s3a, s3n, wasbs, gs. Cloud based stores " +
-      "generally would be total separate from the executors and likely have a higher I/O read " +
-      "cost. Many times the cloud filesystems also get better throughput when you have multiple " +
-      "readers in parallel. This is used with spark.rapids.sql.format.parquet.reader.type")
+      "filesystems. Schemes already included: dbfs, s3, s3a, s3n, wasbs, gs, abfs, abfss. Cloud " +
```

Comment:
Nit: It would be nice to not have to keep these manually in-sync, and also would be nice to list them alphabetically.  For example, moving the default cloud scheme list from `GpuMultiFileReader` to here with something like:
```scala
/** List of schemes that are always considered cloud storage schemes */
val DEFAULT_CLOUD_SCHEMES = Seq("abfs", "abfss", "dbfs", "gs", "s3", "s3a", "s3n", "wasbs")

val CLOUD_SCHEMES = conf("spark.rapids.cloudSchemes")
[...]
    s"filesystems. Schemes already included: ${DEFAULT_CLOUD_SCHEMES.mkString(", ")}. Cloud " +
```

Then GpuMultiFileReader can create its cloud schemes HashSet from the list in RapidsConf.

Reply:
Thx, Done